main/lsp/LSPIndexer.cc

```diff
@@ -75,9 +75,9 @@ void LSPIndexer::computeFileHashes(const vector<shared_ptr<core::File>> &files)
     computeFileHashes(files, *emptyWorkers);
 }
 
-PathType LSPIndexer::getTypecheckingPathInternal(
-    const vector<shared_ptr<core::File>> &changedFiles,
-    const UnorderedMap<core::FileRef, shared_ptr<core::File>> &evictedFiles) const {
+PathType
+LSPIndexer::getTypecheckingPathInternal(const vector<shared_ptr<core::File>> &changedFiles,
+                                        const UnorderedMap<core::FileRef, shared_ptr<core::File>> &evictedFiles) const {
```

Comment:
w.r.t. this "formatting and test fixes" commit, you should be able to use bazel to build `clang-format` and then move it to your PATH and point your editor at it, so that it runs on save. happy to help set this up if you'd like, but it should be as easy as

```shell
./bazel build //tools:clang-format
cp -r bazel-bin/tools/clang-format* ~/.local/bin
```

(or some other folder that is already on your PATH)